cms/djangoapps/contentstore/views/certificates.py

```diff
@@ -300,7 +301,7 @@ def certificate_activation_handler(request, course_key_string):
     # Only global staff (PMs) are able to activate/deactivate certificate configuration
     if not GlobalStaff().has_user(request.user):
         raise PermissionDenied()
-    course_key = CourseKey.from_string(course_key_string)
+    course_key = course_key_from_string_or_404(course_key_string)
```

Comment:
this view returns JSON and only takes `POST` — no reason to use this helper here